### PR TITLE
Newsweek.Com Clone

### PR DESCRIPTION
**Pull request by Eshy10**
## Newsweek.com Clone (Using BootStrap to create a responsive design)
- Added Navigation Bar
- Added Articles
- Added Footer
- Added design for different Screen Sizes
[Live Version](https://rawcdn.githack.com/Eshy10/Newsweek.com-Clone/bff8020d7804707bb3b86d17bba0b4284e446784/index.html)

### CODE REVIEW

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1067 @@
...
+          Choose A Membership That's Perfect for You!
+        </p>
+        <div class="mt-2">
+         <a href="#" class="btn subscribe font-weight-bold rounded-0 text-white">SUBSCRIBE <span>&gt;</span></a>
+        </div>
+       </div>
+     </div>
+     </div>
+     <div class="col-lg-4 col-md-12">
+      <div class="card mt-5" style="max-width: 1000px; max-height:500px;">
```
Comment (pelzolga123) [inline index.html]: No inline styling, please kindly move all styling to style.css

**Comment by pelzolga123** [comment]
## Hi Shola!  
### Your project looks :+1: amazing!
Here are things you need to change, before this project can be approved:
### Tablet version:

* [x] Navigation bar should be aligned like original one:
image

Original:
image

* [x] `Opinion` and `Latest news` sections are not displaying correctly:
image

Original:
image

* [x] Next part should be displayed like original:
image

Original:
image

* [x] `Newsletter` section is missing header
* [x] Footer should look more like an original:
image

Original:
image

### Mobile version:
* [ ] Navbar is messed up a little, it should be fixed:
image

* [ ] First section should be `Top story` right now you have `Featured stories` at the beginning. After `Top story` goes section `More stories`.

* [ ] `Culture and travel` should have same with as other sections:
image

Original:
image

* [ ] `More stories` section should look more like original:
image

Original:
image

* Same for `Options` and `Latest news` sections, they should have bigger width and bigger images:
image
Original:
image

* [ ] Next section should be displayed like original:
image

Original:
image

* [ ] Elements in the next section should be aligned properly:
image

Original:
image

* Footer should look more like original:
image

Original:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by lekegitrepo** [comment]
### Hi, @Eshy10
great job I'm approving this project, but you can check on a few warnings indicated by the validator.
Hence, congratulation you can merge this.
good luck with your next project.

Reviewed by: Temitayo Adeleke
-   Github: [@lekegitrepo](https://github.com/lekegitrepo)
-   Twitter: [@leketemi](https://twitter.com/leketemi)
-   Linkedin: [linkedin](https://www.linkedin.com/in/adeleke-temitayo-a69125188/)

**Review by lekegitrepo — APPROVED** [review]

**Pull request merged**